lib/class-wp-rest-url-details-controller.php

```diff
@@ -191,8 +191,18 @@ public function permissions_check() {
 	 * @return string|WP_Error The HTTP response from the remote URL, or an error.
 	 */
 	private function get_remote_url( $url ) {
+
+		// Provide a modified UA string to workaround web properties which block WordPress "Pingbacks".
+		// Why? The UA string used for pingback requests contains `WordPress/` which is very similar
+		// to that used as the default UA string by the WP HTTP API. Therefore requests from this
+		// REST endpoint are being unintentionally blocked as they are misidentified as pingback requests.
+		// By slightly modifying the UA string, but still retaining the "WordPress" identification (via "WP")
+		// we are able to work around this issue.
+		$modified_user_agent = 'WP(' . get_bloginfo( 'version' ) . ') ' . get_bloginfo( 'url' ) . ': URL details request.';
```

Comment:
I think this can be made a bit more readable. How about this:

```suggestion
		$modified_user_agent = 'WP-URLDetails/' . get_bloginfo( 'version' ) . ' (+' . get_bloginfo( 'url' ) . ')';
```

Inspired by https://api.slack.com/robots and https://developers.facebook.com/docs/sharing/bot/.

Reply:
Thanks for the format suggestion and references provided. These seem logical so I've amended.